### DISCUSSION TITLE
Will OpenFunction support loading functions dynamically?

### DISCUSSION THREAD

**Discussion by ypjin** [OpenFunction/OpenFunction#338] [General]
Hi, It seems that OpenFunction needs to build a container image for a function, much like the way OpenFaaS does it. Is there a plan to support loading a function dynamically into a container like Fission? What's the consideration for choosing the image approach? Or am I missing anything?

Appreciate your responses!

**Comment by benjaminhuo** [comment]
@ypjin We've plans to speed up cold start like what you mentioned to load function directly.

**Comment by ypjin** [comment]
Thanks, @benjaminhuo! That makes sense.
